translator/cmdutil/translatorutil.go

```diff
@@ -31,7 +31,7 @@ const (
 	exitSuccessMessage       = "Configuration validation first phase succeeded"
 )
 
-// Translate from Json map to toml file
+// TranslateJsonMapToTomlFile translates from JSON map to TOML file.
```

Comment:
Minor: if you're making this change, maybe just delete this comment entirely since it adds no value...

Reply:
fixed